.github/workflows/build.yml

```diff
@@ -16,6 +16,8 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
+          - 3.3.6
+          - 3.1.6
```

Comment:
What do you think about adding Ruby 3.2 to the matrix too?

Reply:
Good call

Reply:
Done in [9c3a465](https://github.com/cedarcode/cose-ruby/pull/66/commits/9c3a4650f6509562f55a3a053d073c606c65358c)